.github/workflows/update-node-packages.yml

```diff
@@ -0,0 +1,27 @@
+name: "Update nodePackages"
+on:
+  workflow_dispatch:
+  schedule:
+    - cron: '51 2 * * 0'
+jobs:
+  createPullRequest:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: cachix/install-nix-action@v12
+      - name: Update nodePackages
+        run: |
+          cd pkgs/development/node-packages
+          ./generate.sh
+      - name: Create Pull Request
+        uses: peter-evans/create-pull-request@v3
+        with:
+          commit-message: Update nodePackages
+          committer: GitHub <noreply@github.com>
+          author: ${{ github.actor }} <${{ github.actor }}@users.noreply.github.com>
+          signoff: false
+          branch: update-node-packages
+          delete-branch: true
+          title: 'WIP: Update nodePackages'
+          body: |
+            Automated weekly nodePackages update
```

Comment:
Please add a link to the script that runs this automation in the commit message / the PR message.